src/status_im/data_store/realm/schemas/account/v19/core.cljs

```diff
@@ -89,4 +95,5 @@
   (remove-contact! new-realm "transactor-personal")
   (remove-contact! new-realm "transactor-group")
   (update-commands (juxt :bot :command) owner-command->new-props new-realm "command")
-  (update-commands (juxt :command) console-requests->new-props new-realm "command-request"))
+  (update-commands (juxt :command) console-requests->new-props new-realm "command-request")
+  (update-commands (juxt :command (comp count :prefill)) transactor-requests->new-props new-realm "command-request"))
```

Comment:
Can you explain why this change is necessary?

Reply:
Because I completely forgot that `/request` commands are creating messages of the type "command-request", so we have to migrate them as well.

Reply:
sorry I mean, what is for example `prefill` about? lacking some context

Reply:
So I would naively imagine prefill being a fn of whatever command is and not a standalone thing, but that evidently doesn't appear to be the case.

Reply:
:prefill is array of prefilled arguments, we distinguish between group command/personal command based on it's size